### DISCUSSION TITLE
Unable to block website for a client tag via important modifier

### DISCUSSION THREAD

**Discussion by gmichels** [AdguardTeam/AdGuardHome#4719] [Q&A]
Hello all,

I am trying to block m.youtube.com for a specific client tag and at the same time enforce restricted mode (strict restricted - restrict.youtube.com, not the moderate restricted which happens when enabling safe search).

I have these in my custom filtering rules:

```
# block mobile YouTube to prevent usage via Safari
||m.youtube.com^$important,ctag=user_child

# use restricted mode for YouTube
www.youtube.com^$dnsrewrite=restrict.youtube.com
m.youtube.com^$dnsrewrite=restrict.youtube.com
youtubei.googleapis.com^$dnsrewrite=restrict.youtube.com
youtube.googleapis.com^$dnsrewrite=restrict.youtube.com
www.youtube-nocookie.com^$dnsrewrite=restrict.youtube.com
```

If I remove the `dnsrewrite` line for m.youtube.com, the rule works as expected. Although I have the `important` modifier, the `dnsrewrite` rule seems to be taking precedence and the requests are going through.

Any ideas on how to accomplish this?

Thank you

**Comment by fernvenue** [comment]
Can be reproduced, I think maybe this is a bug, just created an issue https://github.com/AdguardTeam/AdGuardHome/issues/4723 for this.
